src/udp_stream.cpp

```diff
@@ -6,11 +6,11 @@
 
 #include "rtl_airband.h"
 
-bool raw_stream_init(raw_stream_data *sdata) {
+bool udp_stream_init(udp_stream_data *sdata, mix_modes mode) {
 	// Create the send socket
 	sdata->send_socket = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
```

Comment:
This isn't pretty, since it doesn't allow the user to provide a DNS name and it supports IPv4 only. I suggest using `getaddrinfo(3)` to determine the family, socktype and protocol. See https://github.com/szpajder/dumpvdl2/blob/master/src/output-udp.c for an example.

Reply:
Will take a look.

DNS lookup would only happen at startup (ie in `udp_stream_init()`) and not periodically, correct?  So if a hostname to IP mapping changes the program would need to be restarted?

Reply:
Correct.